tests/Feature/Security/OrderPricingValidationTest.php

```
@@ -0,0 +1,269 @@
+        $sellerRole = Role::create(['name' => 'Seller', 'guard_name' => 'web']);
+        $sellerRole->givePermissionTo(Permission::findOrCreate('create order', 'web'));
+        $sellerRole->givePermissionTo(Permission::findOrCreate('update order', 'web'));
+        $sellerRole->givePermissionTo(Permission::findOrCreate('read order', 'web'));
+
+        Role::findOrCreate('SuperAdmin', 'web')->givePermissionTo([
+            Permission::findOrCreate('create order', 'web'),
+            Permission::findOrCreate('update order', 'web'),
+            Permission::findOrCreate('override order price', 'web'),
+        ]);
+
+        $this->company = Company::factory()->create();
```

PHPStan (2.2.7)

[error] Property Tests\Feature\Security\OrderPricingValidationTest::$company (App\Models\Company) does not accept Illuminate\Database\Eloquent\Collection<int, Illuminate\Database\Eloquent\Model>|Illuminate\Database\Eloquent\Model.


(assign.propertyType)